backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlRepository.java

```diff
@@ -255,11 +287,71 @@ else if (po.isDynAttributeTrue(DYNATTR_TrlUpdateMode_UpdateIdenticalTrls))
 
 		final int updatedCount = DB.executeUpdateEx(sql.toString(), ITrx.TRXNAME_ThreadInherited);
 		logger.debug("Updated {} translation records for {}", updatedCount, po);
+		final CacheMgt cacheMgt = CacheMgt.get();
+		cacheMgt.reset(toTrlTableName(tableName), po.get_ID());
 
-		//
 		return updatedCount >= 0;
 	}
 
+	public void updateBaseTableOnTranslationChange(@NonNull final PO po)
+	{
+		final POTrlInfo trlInfo = po.getPOInfo().getTrlInfo();
+		if (!trlInfo.isTranslations())
+		{
+			return;
+		}
+
+		final String trlTableName = po.get_TableName();
+		final String baseTableName = toBaseTableName(trlTableName);
+		if(DB.isDBColumnPresent(baseTableName, COLUMNNAME_AD_Element_ID))
```

Comment:
the whole purpose of building up metadata like POTrlInfo is to not have the need to do more calculations each time, like:
* checking if the column exist
* toBaseTableName(trlTableName) ... if u need this info.. IMHO you shall have the baseTableName and trlTableName right in the POTrlInfo.

Reply:
this check doesn't work and will be replaced. I wanted to not update base_tables that get updated by AD_Element, but not all tables have this column (for example AD_Field)

Reply:
still pending